OsmAnd/res/values/phrases.xml

```diff
@@ -1820,9 +1820,9 @@
 
 	<string name="poi_public_bath">Public bath</string>
 
-	<string name="poi_male_yes">Male</string>
+	<string name="poi_male_yes">Male Only</string>
```

Comment:
I don't think this is correct cause there are toiltets with male/yes + female/yes

Reply:
The OSM wiki actually uses the strict word "only" as well. But as mentioned, multiple gender-keys can be `yes` on one POI.
https://wiki.openstreetmap.org/wiki/Key:male

For an app like OSMAnd, I suggest: Male allowed / Female allowed